pages/whats-new/2020-12-05-redis.mdx

```diff
@@ -0,0 +1,10 @@
+---
+title: "Grouparoo can run locally without Redis"
+date: "2020-12-05"
+tags: ["new"]
+description: "You can now run Grouparoo locally without needing to run Redis"
+image: "releases/2020-11-17-sqlite.png"
+github: "https://github.com/grouparoo/grouparoo/pull/993"
+---
+
+You can now run Grouparoo locally without needing to run Redis. In your `.env` file, set your redis URL to `REDIS_URL="redis://mock"`. If you choose to run Grouparoo this way, you can't use Grouparoo in a cluster. Additionally there is no data persistence, so if you stop and restart your server, there might be data loss.
```

Comment:
> if you stop and restart your server, there *will* be data loss.

I'd add something like:

> This feature makes it possible for you too try out Grouparoo without needing any dependencies!

And maybe:

> This is an important part of the foundation for Grouparoo v0.2.  Learn more here https://github.com/grouparoo/grouparoo/discussions/1054

Reply:
Thanks!